monitoring/configure-metrics-tools.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x # echo so that users can understand what is happening
+set -e # exit on error
+
+ctx="kind-kubeflex"
+wds="wds1"
+
+if [ "$1" == "--space-name" ]; then
+    ctx="$2"
+    shift 2
+fi
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+
+# set core context
+kubectl config use-context $ctx
+
+: --------------------------------------------------------------------
+: Configure kubestellar controller for prometheus scraping
+: --------------------------------------------------------------------
+: 1. configure the rbac for the kube-rbac-proxy 
```

Comment:
I would add "in the KubeStellar controller-manager Pod".

Reply:
Fixed!